### ISSUE TITLE
To do 5/9

### ISSUE THREAD

**Issue by xuanyuzhou98** [xuanyuzhou98/OnDeviceTrain#9]
Preliminary milstones:
#milestone 1: +batch norm + train cifar and check accuracy
#milestone 2: train imagenet, 50 accuracy, 1 week training time, 50 + accuracy

Potential directions for applications:
1.  text completion, autoreply - finetune(benchmark? energy? total flops?)
2. customized image filter(style transfer), editing, **tagging** - maybe generative?
3. audio style transfer 
4. improve camera itself

To do:
1. image tagging - pick small datasets, do image classification, image style transfer
2. audio - review wavenet paper, audio style transfer
